src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -465,5 +465,95 @@ public void SendMail_DeliveryFormat_SubjectEncoded(bool useAsyncSend, bool useSe
                 server.Stop();
             }
         }
+
+        [Fact]
+        public void UseDefaultCredentialsUnset_ShouldReturn_CredentialsAsNull()
```

Comment:
Some comments about naming tests:

```
<property or method under test>_<conditions>_<expected result>"
```

So, this test should named `Credentials_ctor_Null`.  This particular test is testing the value of the .Credentials property and not the .UseDefaultCredentials property.

Reply:
So, please consider this pattern in the tests below and revise accordingly.

Reply:
Hi, thanks, I will do so